src/platform/site-wide/user-nav/tests/containers/Main.unit.spec.jsx

```diff
@@ -87,6 +88,27 @@ describe('<Main>', () => {
       wrapper.unmount();
     });
 
+    it('should automatically initiate a session log in if a SSOe-flagged user has an active SSOe session but no vets-website session', () => {
+      localStorage.setItem('hasSessionSSO', true);
+      props.useSSOe = true;
+      authUtils.autoLogin = sinon.spy();
+      sessionStorage.setItem = sinon.spy();
```

Comment:
Should this be handled similarly to the setup of `global.window`?

Reply:
just tested again with this commented out and it passed. I think this was needed before I correctly imported `authUtils` to spy on `autoLogin`, in which case you're right, but it can be removed completely now